qt/DetailsDialog.cc

```diff
@@ -1017,8 +997,9 @@ DetailsDialog::onIdleModeChanged (int index)
 }
 
 void
-DetailsDialog::onIdleLimitChanged ()
+DetailsDialog::onIdleLimitChanged (int unused)
 {
+    Q_UNUSED(unused);
```

Comment:
Personally I dislike this. If the argument isn't used, C++ allows to just omit its name in argument list, so you could write it as e.g. `DetailsDialog::onIdleLimitChanged (int /*index*/)`